### PR TITLE
Update Index.html

### PR DESCRIPTION
**Pull request by bear217**
started the basic framework for the hack club website, worked to create a basic page layout that can be expanded more.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,2 +1,127 @@
...
+<body>
+  <h1>P2P Hack Club</h1>
+  <hr>
+  <div class="gridcontainer">
+    <div class="grid-item"><a class="main" href="https://google.com">About Hack Club</a></div>
+    <div class="grid-item"><a class="main" href="https://google.com">Meeting Schedule</a></div>
+    <div class="grid-item"><a class="main" href="https://google.com">Ongoing Projects</a></div>
+  </div>
+  <h1 style="font-size:30px;text-align:left">Introduction to GitHub:</h1>
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/w3jLJU7DT5E" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (thatrobotdev) [inline index.html:112]: Nice video :D

File: index.html [hunk 2]
```diff
@@ -1,2 +1,127 @@
...
+  <br>
+  <h1 style="font-size:30px;text-align:left">Events:</h1>
+  <div class="events">
+    <h2>HacktoberFest</h2>
```
Comment (thatrobotdev) [inline index.html:116]:
```suggestion
    <h2>Hacktoberfest</h2>
```

File: index.html [hunk 3]
```diff
@@ -1,2 +1,127 @@
...
+    <hr>
+    Hacktoberfest is an annual event that takes place every october in the spirit of open source and collaboration. For more information, Click <a class="hyperlink" href="https://hacktoberfest.digitalocean.com">Here!</a>
```
Comment (thatrobotdev) [inline index.html:118]:
```suggestion
    Hacktoberfest is an annual event that takes place every October in the spirit of open source and collaboration. For more information, Click <a class="hyperlink" href="https://hacktoberfest.digitalocean.com">Here!</a>
```

**Review by thatrobotdev — CHANGES_REQUESTED** [review]
I love it! I requested some small changes, but you've really started the website well! I'm going to go in later this week hopefully and add some of the content that we've had in previous years :D 🦜

**Comment by bear217** [comment]
I committed the changes you requested! I should be able to commit some more stuff in a bit too.

**Comment by thatrobotdev** [comment]
Looks great, thank you so much! 🎃🎃

**Pull request merged**